etc/config/cuda.amazon.properties

```diff
@@ -144,16 +144,36 @@ compiler.cltrunk.exe=/opt/compiler-explorer/clang-trunk/bin/clang++
 compiler.cltrunk.demangler=/opt/compiler-explorer/gcc-snapshot/bin/c++filt
 compiler.cltrunk.options=--gcc-toolchain=/opt/compiler-explorer/gcc-snapshot --cuda-path=/opt/compiler-explorer/cuda/11.3.1 --cuda-gpu-arch=sm_86 --cuda-device-only -Wno-unknown-cuda-version
 
-group.hipclang.compilers=hiptrunk
+group.hipclang.compilers=hiptrunk:clang-rocm-trunk:clang-rocm-40502:clang-rocm-50002:clang-rocm-50103:clang-rocm-50203
 group.hipclang.isSemVer=true
-group.hipclang.baseName=clang (HIP)
+group.hipclang.baseName=clang
 group.hipclang.compilerType=clang-hip
 group.hipclang.objdumperType=llvm
-group.hipclang.options=-x hip --offload-arch=gfx908 --cuda-device-only -nocudainc -nocudalib -include __clang_hip_runtime_wrapper.h
 
-compiler.hiptrunk.semver=trunk
+compiler.hiptrunk.semver=trunk (w/o device libs)
 compiler.hiptrunk.exe=/opt/compiler-explorer/clang-trunk/bin/clang++
 compiler.hiptrunk.objdumper=/opt/compiler-explorer/clang-trunk/bin/llvm-objdump
+compiler.hiptrunk.options=-x hip --offload-arch=gfx908 --cuda-device-only -mllvm -amdgpu-early-inline-all=true -mllvm -amdgpu-function-calls=false -O3 -D__HIP_PLATFORM_AMD__=1 -D__HIP_PLATFORM_HCC__=1 -nocudainc -nocudalib -include __clang_hip_runtime_wrapper.h
+compiler.clang-rocm-trunk.exe=/opt/compiler-explorer/clang-rocm-trunk/bin/clang++
```

Comment:
Only bad thing about this is that compilers won't be ordered by semver using this prefix scheme. I have some ideas to fix it that will try to implement once this gets merged :)